dynamic_rest/serializers.py

```diff
@@ -407,6 +445,26 @@ def to_representation(self, instance):
             embed=self.embed
         )
 
+    def to_internal_value(self, data):
+        value = super(WithDynamicSerializerMixin, self).to_internal_value(data)
+        id_attr = getattr(self.Meta, 'update_lookup_field', 'id')
+        request_method = getattr(
+            getattr(self.context.get('view'), 'request'),
```

Comment:
FYI, DRF puts the request in context [by default](https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/generics.py#L136).